src/components/map/hooks/use-realtime-map-enabled.ts

```diff
@@ -0,0 +1,17 @@
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {StorageModelKeysEnum} from '@atb/storage';
+import {useDebugOverride} from '@atb/debug';
+
+export const useRealtimeMapEnabled = () => {
+  const [debugOverride, setDebugOverride] = useRealtimeMapDebugOverride();
+  const {enable_realtime_map: enabledInRemoteConfig} = useRemoteConfig();
+
+  if (debugOverride !== undefined) {
+    return [debugOverride, setDebugOverride];
```

Comment:
Wondering about this `return [debugOverride, setDebugOverride]` here. This means the return type for `useRealtimeMapEnabled` will be `[boolean, (boolean) => void] | boolean`?

Reply:
Oups, had some uncommited changes